registry/handlers/layerupload.go

```diff
@@ -161,11 +163,45 @@ func (luh *layerUploadHandler) GetUploadStatus(w http.ResponseWriter, r *http.Re
 	w.WriteHeader(http.StatusNoContent)
 }
 
-// PutLayerUploadComplete takes the final request of a layer upload. The final
-// chunk may include all the layer data, the final chunk of layer data or no
-// layer data. Any data provided is received and verified. If successful, the
-// layer is linked into the blob store and 201 Created is returned with the
-// canonical url of the layer.
+// PutLayerData writes data to an upload.
+func (luh *layerUploadHandler) PutLayerData(w http.ResponseWriter, r *http.Request) {
```

Comment:
I almost made this change but didn't seem there was necessarily a consistent theme, Post -> Start, Delete->Cancel. I will make this change to make it more clear.